test/tests/vectorpostprocessors/material_vector_postprocessor/basic.i

```diff
@@ -4,6 +4,7 @@
   dim = 2
   nx = 10
   ny = 10
+  allow_renumbering = false
```

Comment:
I guess I'm a little concerned that this is now required for this test whereas it wasn't before?

Reply:
Leftover from debugging and may not be required (though doesn't seem like a bad idea considering element numbers are in the gold file). I'll investigate.

Reply:
No, I think this is a real issue that we stumbled upon. The element numbering is different between replicated and distributed mesh in this test, but constant material properties are assigned for all elements, so the element number/material property pairs are unchanged for different numbering schemes even though element N is different. Including element coordinates exposed this difference.

Reply:
👍